<ui version="4.0" >
 <class>SvnLogviewWidget</class>
 <widget class="QWidget" name="SvnLogviewWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Logview</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="indentation" >
       <number>-7</number>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListView" name="listView" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="3" />
 <resources/>
 <connections/>
</ui>
